databricks-geococoding/README.md

```diff
@@ -0,0 +1,41 @@
+![Precisely](../Precisely_Logo.png)
+
+# Databricks Geocoding Sample
+This sample for Databricks demonstrates how to install, configure, and execute against the appropriate geocoding spark driver. 
```

Comment:
This sample for Databricks demonstrates how to install, configure, and execute a geocoding process in Databricks.

Reply:
done